pages/index.jsx

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+import Head from "next/head";
+import { NavBar } from "../src/components/Nav/NavBar";
+import { useState } from "react";
+import ReactPlayer from "react-player";
+import { LogoImage } from "../src/components/Header/Logo";
+
+export const Home = () => {
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  return (
+    <>
+      <Head>
+        <title>Corporate Campus Configurator</title>
+      </Head>
+      <LogoImage />
+      <NavBar />
+      <h1>CORPORATE CAMPUS CONFIGURATOR</h1>
```

Comment:
You could consider using css styles (`text-transform: uppercase;`) to get uppercase text. It is still perfectly fine as you're doing it.